lib/assessment.js

```diff
@@ -52,7 +82,17 @@ var inject;
 // store the function in the variable named max
 
 /* your code starts here */
-var max;
+var max = function(x) {
+  var n = arguments[0];
+  
+  for (var i = 0; i < arguments.length; i++) {
```

Comment:
Good point - since you're comparing arguments[0] with itself that first time, you don't actually need to start the `for` loop at 1.